shaders.go

```diff
@@ -244,14 +258,14 @@ func (s *basicShader) generateBufferContent(ren *RenderComponent, buffer []float
 
 	var changed bool
 
-	//setValue(buffer, 0, 0, &changed)
-	//setValue(buffer, 1, 0, &changed)
+	//setBufferValue(buffer, 0, 0, &changed)
+	//setBufferValue(buffer, 1, 0, &changed)
```

Comment:
I'm actually keeping them, because they show what the value is of the buffer at each index location. 

The code is commented for performance reasons, the code remains there (commented) for us to see what's going on.